tools/fast_puller_.py

```diff
@@ -58,6 +65,12 @@ def main():
 
   transport = transport_pool.Http(httplib2.Http, size=_THREADS)
 
+  if args.certificates:
+    for item in args.certificates:
+      logging.info('Adding certificate %s', item)
+      key, cert, domain = item.split(',')
+      transport.add_certificate(key, cert, domain)
```

Comment:
Can you please add a test to ensure, this actually works. Looking at the code I have some doubts.